[2024-12-18T17:47:20.686+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_pipeline.data_processing manual__2024-12-18T17:47:16.108081+00:00 [queued]>
[2024-12-18T17:47:20.696+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_pipeline.data_processing manual__2024-12-18T17:47:16.108081+00:00 [queued]>
[2024-12-18T17:47:20.696+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2024-12-18T17:47:20.718+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): data_processing> on 2024-12-18 17:47:16.108081+00:00
[2024-12-18T17:47:20.725+0000] {standard_task_runner.py:60} INFO - Started process 128 to run task
[2024-12-18T17:47:20.728+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'data_pipeline', 'data_processing', 'manual__2024-12-18T17:47:16.108081+00:00', '--job-id', '62', '--raw', '--subdir', 'DAGS_FOLDER/data_pipeline_dag.py', '--cfg-path', '/tmp/tmpqp8aiyie']
[2024-12-18T17:47:20.731+0000] {standard_task_runner.py:88} INFO - Job 62: Subtask data_processing
[2024-12-18T17:47:20.790+0000] {task_command.py:423} INFO - Running <TaskInstance: data_pipeline.data_processing manual__2024-12-18T17:47:16.108081+00:00 [running]> on host 1d3bd1c7da18
[2024-12-18T17:47:20.887+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='data_pipeline' AIRFLOW_CTX_TASK_ID='data_processing' AIRFLOW_CTX_EXECUTION_DATE='2024-12-18T17:47:16.108081+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-18T17:47:16.108081+00:00'
[2024-12-18T17:47:21.010+0000] {logging_mixin.py:188} INFO - Data processed
[2024-12-18T17:47:21.011+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-12-18T17:47:21.021+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=data_pipeline, task_id=data_processing, execution_date=20241218T174716, start_date=20241218T174720, end_date=20241218T174721
[2024-12-18T17:47:21.061+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-12-18T17:47:21.082+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
